Gemfile

```diff
@@ -2,7 +2,7 @@ source "https://rubygems.org"
 gemspec
 
 group :development, :test do
-  gem "omnibus", git: "git://github.com/datadog/omnibus-ruby.git", branch: "datadog-5.5.0"
+  gem "omnibus", git: "https://github.com/datadog/omnibus-ruby.git", branch: "datadog-5.5.0"
```

Comment:
you might also want to change all http to https :)

Reply:
Done !